src/ontology/cl-edit.owl

```diff
@@ -21670,31 +21675,34 @@ AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004231 "https
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0004231 "cell")
 AnnotationAssertion(oboInOwl:id obo:CL_0004231 "CL:0004231")
 AnnotationAssertion(rdfs:label obo:CL_0004231 "recurving diffuse amacrine cell")
-SubClassOf(obo:CL_0004231 obo:CL_0000561)
+SubClassOf(obo:CL_0004231 obo:CL_0004251)
 SubClassOf(obo:CL_0004231 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0008923))
 SubClassOf(obo:CL_0004231 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0008924))
 SubClassOf(obo:CL_0004231 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0008925))
 
 # Class: obo:CL_0004232 (starburst amacrine cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:9620701") obo:IAO_0000115 obo:CL_0004232 "An amacrine cell with a flat dendritic arbor and a medium dendritic field. Starburst amacrine cells have post-synaptic terminals in S2.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:9620701") obo:IAO_0000115 obo:CL_0004232 "An amacrine cell with a flat dendritic arbor and a medium dendritic field. Starburst amacrine cells have post-synaptic terminals in S2. This cell type releases the neurotransmitters gamma-aminobutyric acid and acetilcholine.")
```

Comment:
1- recommend to add (GABA) after gamma-aminobutyric acid to aid searching by text.
2 acetilcholine --> acetylcholine

Reply:
I can apply the same (for GABA) in all other GABAergic amacrine cells.